modules/terraform/format.go

```diff
@@ -167,3 +199,13 @@ func primitiveToHclString(value interface{}, isNested bool) string {
 		return fmt.Sprintf("%v", v)
 	}
 }
+
+// Check whether string exists in a slice
+func sliceContains(s []string, e string) bool {
```

Comment:
NIT: Use https://github.com/gruntwork-io/terratest/blob/master/modules/collections/lists.go#L32 instead.

Reply:
Thanks @brikis98, I've updated the code to use ListContains instead of sliceContains

Reply:
Don't forget to remove the now unused `sliceContains` method!

Reply:
Sorry, thanks for pointing it out, it's been removed now!